src/Layout/Presentation/SingleParty/SingleParty.tsx

```diff
@@ -0,0 +1,271 @@
+                totalSeats: partyResult.totalSeats,
+                marginInVotes: getMarginInVotes(partyCode, vulnerableMap, vulnerableVotes, this.props.firstDivisor),
+                seatWinner: vulnerableVotes?.winner.partyCode === partyCode,
+                closestVotes: vulnerableVotes?.partyCode === partyCode,
+                quotientMargin: quotientMap?.get(partyCode) || 0,
+                closestQuotient: vulnerable?.runnerUp?.partyCode === partyCode,
+                proportionality: partyResult.proportionality
+            };
+        } else {
+            return {
+                district: districtResult.name,
+                votes: 0,
+                percentVotes: 0,
+                districtSeats: 0,
+                levelingSeats: 0,
+                totalSeats: 0,
+                marginInVotes: getMarginInVotes(partyCode, vulnerableMap, vulnerableVotes, this.props.firstDivisor),
+                seatWinner: vulnerableVotes?.winner.partyCode === partyCode,
+                closestVotes: vulnerableVotes?.partyCode === partyCode,
+                quotientMargin: quotientMap?.get(partyCode) || 0,
+                closestQuotient: vulnerable?.runnerUp?.partyCode === partyCode,
+                proportionality: 0
+            };
+        }
+    }
+
+    render() {
+        const partyCodeList = this.props.partyCodeList;
+        const partyCode = partyCodeList.indexOf(this.props.partySelected) >= 0 ? this.props.partySelected : partyCodeList[0];
+
+        const calculateVulnerable =
+        isQuotientAlgorithm(this.props.algorithm) && this.props.districtSeats > 0;
+        const data = this.props.districtResults.map((districtResult) => this.constructSinglePartyResult(districtResult, calculateVulnerable, partyCode));
+        const decimals = this.props.decimals;
+        const proportionalities = data.map((value) => value.proportionality);
+        let label: string;
+        let index: number;
+        switch (this.props.disproportionalityIndex) {
+            case DisproportionalityIndex.LOOSEMORE_HANBY: {
+                label = "L-H";
+                index = proportionalities.map((value) => Math.abs(value)).reduce(toSum, 0) / 2;
+                break;
+            }
+            case DisproportionalityIndex.GALLAGHER: {
+                label = "LSq";
+                index = Math.sqrt(proportionalities.map((value) => value * value).reduce(toSum, 0) / 2);
+                break;
+            }
+            default: {
+                checkExhaustively(this.props.disproportionalityIndex);
+                label = "Error";
+                index = -1;
+            }
+        }
+        return (
+            <React.Fragment>
+                <PartySelect
+                    selectParty={this.props.selectParty}
+                    partyCodeList={partyCodeList}
+                    partySelected={this.props.partySelected}
+                    partyMap={this.props.partyMap}
+                />
+                <ReactTable
```

Comment:
We might want to put some kind of "legend" above the ReactTable for explaining the trophies and whatnot, as there often being many of them (and it is a bit unclear what they mean in such cases).